lazycatlabs/blog.lazycatlabs
# feat: use catpuccin theme

## What it does

- use catpuccin theme for tailwind
- change the layout from List to Grid for Articles
- update layout if search not found

## How to test

1. open the web
2. observe the view
3.

## Screenshot
![image](https://github.com/Lzyct/blog.lazycatlabs/assets/1531684/750debfb-dc82-4c2b-a577-916cfe8ca965)

![image](https://github.com/Lzyct/blog.lazycatlabs/assets/1531684/a8114f20-b100-45b7-bf15-e3bc3bbd3bdf)
